functions-and-scope.js

```diff
@@ -72,12 +179,59 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+console.log( "===========================  OPDRACHT 3a ============================" )
+
+let gradesExtra = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
+// Declareer een variabel en stel deze in op het eerste element van de array. 
+let highestGrade = gradesExtra[0];
```

Comment:
Heel goed dat je de eerste waarde van de array pakt in plaats van `0` bijvoorbeeld! 👍